### ISSUE TITLE
Timecodes for "ML Zoomcamp 6.8 - XGBoost Parameter Tuning"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#168]
Youtube video: https://www.youtube.com/watch?v=VX6ftRzYROM

**Comment by dimzachar** [comment]
0:00:00 - Lesson 8: Tuning parameters for extreme boost model
0:02:01 - Creating and plotting XGBoost evaluation scores.
0:04:28 - Understanding the impact of learning rate.
0:06:39 - Comparing model performance with different parameters.
0:08:30 - Tuning max depth parameter for model training.
0:10:32 - Zoom in to analyze tree performance.
0:12:33 - Tuning child weight for decision tree.
0:14:32 - Training final model with 175 iterations.
0:16:26 - Experimenting with parameter values for optimization.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
